Makefile

```diff
@@ -1,5 +1,5 @@
 test:
-	@(cd pystarport/tests && poetry run pytest)
+	@(cd pystarport/tests && ./test_expansion/generate-test-yamls && poetry run pytest)
```

Comment:
I see, you converted the yaml to jsonnet and generate it on the fly, what's the reason of doing this?

Reply:
The #50 is more or less fixed by yaml include support I think, this PR is mainly to add more expressiveness by support jsonnet.